docs/product-editor-development/common-tasks.md

```diff
@@ -0,0 +1,31 @@
+# Common tasks
+
+> ⚠️ **Notice:** This documentation is currently a **work in progress**. Please be aware that some sections might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
+
+## Adding a group/section/field next to an existing one
+
+Here's a snippet that adds a new block to the catalog section for simple products, between the first and second fields (order 15):
+
+```php
+if ( ! function_exists( 'YOUR_PREFIX_add_block_after_categories' ) ) {
+	/**
+	 * Add a new block to the template.
+	 */
+	function YOUR_PREFIX_add_block_after_categories( BlockInterface $product_categories_field ) {
+    $product_categories_field->get_parent()->add_block(
+      [
+        'id'         => 'your-prefix-id',
+        'blockName'  => 'your-block-name',
+        'order'      => $product_categories_field->get_order() + 5,
+        'attributes' => [
+          'key'   => 'value',
+        ],
+      ]
+    );
+	}
+}
+add_action( 'woocommerce_block_template_area_product-form_after_add_block_product-categories', 'YOUR_PREFIX_add_block_after_categories' );
+```
```

Comment:
As with your comment on my PR, I think that the `using` statement for `BlockInterface` should be added.

Also, I wonder if an actual block name should be used in the example. It would be nice it the code could be copied into a plugin and it would just work!